### PR TITLE
Ocis cs3 ldap compose

### PR DESCRIPTION
**Pull request by IljaN**
Requires #1020

### CODE REVIEW

File: deployments/examples/cs3_users_ocis/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+document this deployment example in docs/ocis/deployment/owncloud10_with_ocis_web.md
```
Comment (wkloucek) [inline deployments/examples/cs3_users_ocis/README.md]: please change to:
```
document this deployment example in: docs/ocis/deployment/cs3_users_ocis.md
```

File: deployments/examples/cs3_users_ocis/README.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+---
+
+Please refer to [our documentation](https://owncloud.github.io/ocis/deployment/owncloud10_with_ocis_web/)
```
Comment (wkloucek) [inline deployments/examples/cs3_users_ocis/README.md]: Should be
```
Please refer to [our documentation](https://owncloud.github.io/ocis/deployment/cs3_users_ocis/)
```

But therefore documentation needs to be added ;-)

**Review by wkloucek — CHANGES_REQUESTED** [review]

File: deployments/examples/cs3_users_ocis/ocis/config/proxy-config.json [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+          "endpoint": "/index.php/",
+          "backend": "http://localhost:9140"
+        },
+        {
+          "endpoint": "/data",
+          "backend": "http://localhost:9140"
+        },
+        {
+          "endpoint": "/api/v0/accounts",
+          "backend": "http://localhost:9181"
```
Comment (wkloucek) [inline deployments/examples/cs3_users_ocis/ocis/config/proxy-config.json:69]: How does this play together with `PROXY_ACCOUNT_BACKEND_TYPE` ?

Comment (wkloucek) [reply deployments/examples/cs3_users_ocis/ocis/config/proxy-config.json:69]: Did you change something on the proxy-config.json compared to the default configuration?

Comment (IljaN) [reply deployments/examples/cs3_users_ocis/ocis/config/proxy-config.json]: Yes, the ocs regex route is different. Removed references to accounts service now.

**Comment by IljaN** [comment]
Seeing Following error:
```
 ERR error starting the grpc server error="unable to register services: rgrpc: grpc service usershareprovider could not be started,: error loading the file containing the shares: error opening/creating the file: /var/tmp/ocis/shares.json: open /var/tmp/ocis/shares.json: no such file or directory" service=storage
```

**Review by wkloucek — APPROVED** [review]
needs to tested again when #1020 is merged, but generally ok

File: deployments/examples/cs3_users_ocis/docker-compose.yaml [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+      - "traefik.http.routers.ocis-secure.service=ocis"
+      - "traefik.http.services.ocis.loadbalancer.server.port=9200"
+    restart: always
+
+  ldap-server:
+    image: osixia/openldap:latest
+    command: --copy-service --loglevel debug
+    environment:
+      LDAP_TLS_VERIFY_CLIENT: never
+      LDAP_DOMAIN: owncloud.test
```
Comment (wkloucek) [inline deployments/examples/cs3_users_ocis/docker-compose.yaml:117]: must also be owncloud.com

**Pull request merged**